### PR TITLE
New package FortranFiles.jl

### PR DESCRIPTION
**Pull request by traktofon**
Hello,
the package "FortranFiles.jl" is for conveniently reading and writing Fortran "unformatted" (i.e. binary) files. Documentation is already there, and automatic testing is set up and working. Please see the package's Github page for details. Thank you!

### CODE REVIEW

File: FortranFiles/versions/0.0.1/requires [hunk 1]
```diff
@@ -0,0 +1,2 @@
+julia 0.5
+Compat
```
Comment (tkelman) [inline FortranFiles/versions/0.0.1/requires]: It's best to declare a minimum version dependency that implements the features you need.

Comment (tkelman) [inline FortranFiles/versions/0.0.1/requires]: you can edit this pull request in place in this branch of your metadata fork

Comment (tkelman) [inline FortranFiles/versions/0.0.1/requires]: if `@compat abstract type` is the only thing you're using, that was in 0.17

**Comment by traktofon** [comment]
Thanks @tkelman !

**Pull request merged**